shared/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@aragon/templates-shared",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Shared helpers for Aragon templates",
+  "dependencies": {
+    "@aragon/apps-agent": "1.0.0",
+    "@aragon/apps-finance": "3.0.0",
+    "@aragon/apps-shared-migrations": "1.0.0",
+    "@aragon/apps-shared-minime": "1.0.0",
+    "@aragon/apps-token-manager": "2.1.0",
+    "@aragon/apps-vault": "4.1.0",
+    "@aragon/apps-voting": "2.1.0",
```

Comment:
What do you think about not pinning these app dependencies, but using a `^`?

Since this is shared, we should use a looser range and let the actual templates pick which version of the apps they want to use. `@aragon/id` and `@aragon/os` should be outside the scope of other templates so we can pin them.

Originally we were going to set up something like dependabot via these loose versions to make sure the dao-templates still worked with new versions of any dependent apps. I think that still makes sense, and in production deployment commits we can either pin or include a `package-lock.json`.